File: Service/Implement/DanhMucHinhThucNuoiService.cs
namespace Service.Implement
{
    public class DanhMucHinhThucNuoiService : BaseService<DanhMucHinhThucNuoi, IDanhMucHinhThucNuoiRepository>
    , IDanhMucHinhThucNuoiService
    {
    private readonly IDanhMucHinhThucNuoiRepository _DanhMucHinhThucNuoiRepository;
    public DanhMucHinhThucNuoiService(IDanhMucHinhThucNuoiRepository DanhMucHinhThucNuoiRepository) : base(DanhMucHinhThucNuoiRepository)
    {
    _DanhMucHinhThucNuoiRepository = DanhMucHinhThucNuoiRepository;
    }
    }
    }

File: CMS/src/app/shared/DanhMucHinhThucNuoi.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucHinhThucNuoi } from 'src/app/shared/DanhMucHinhThucNuoi.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucHinhThucNuoiService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucHinhThucNuoi";
    }
}

